src/test/java/runge/wordledictionary/WordleGameTest.java

```diff
@@ -0,0 +1,46 @@
+package runge.wordledictionary;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.CharArrayReader;
+import java.io.IOException;
+import java.util.Arrays;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class WordleGameTest
+{
+    @Test
+    public void WordleGameCorrectTest() throws IOException
+    {
+
+        //given
+        WordleGame game1 = new WordleGame();
+
+        //when
+        String correctWord = game1.getRandomWord();
+        CharResult[] correctArray = {CharResult.Correct, CharResult.Correct, CharResult.Correct, CharResult.Correct, CharResult.Correct};
+        CharResult[] status = game1.guess(correctWord);
+
+        //then
+        assertArrayEquals(correctArray, status);
+
+    }
+
+    @Test
+    public void WordleGameIncorrectTest() throws IOException
+    {
+        //given
+        WordleGame game1 = new WordleGame();
+
+        //when
+        CharResult[] notCorrect = game1.guess("xxxxx");
+        CharResult[] correctArray = {CharResult.Correct, CharResult.Correct,
+                CharResult.Correct, CharResult.Correct, CharResult.Correct};
+
+        boolean returnStatus = Arrays.equals(notCorrect,correctArray);
+
+        //then
+        assertFalse(false, String.valueOf(returnStatus));
```

Comment:
What does the "false" parameter in assertFalse do?

You can do all of this in one line.

Reply:
false essentially passes the test. I set my boolean returnStatus, if it equals false then I know my test passed because the value I put in should never work.

Reply:
I figured it out